code/game/objects/items/weapons/cigs_lighters.dm

```diff
@@ -500,16 +518,20 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 	slot_flags = SLOT_BELT
 	attack_verb = list("burnt", "singed")
 	var/base_state
-	var/activation_sound = 'sound/items/lighter_on.ogg'
-	var/desactivation_sound = 'sound/items/lighter_off.ogg'
+	var/activation_sound = list(
```

Comment:
Where-ever this is used, you need to do `pick(activation_sound)` instead of just `activation_sound` now.